specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -316,6 +316,151 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Databricks/workspaces/{workspaceName}/virtualNetworkPeerings/{peeringName}": {
+      "get": {
+        "tags": [
+          "vNetPeering"
+        ],
+        "operationId": "vNetPeering_Get",
+        "description": "Gets the workspace vNet Peering.",
+        "x-ms-examples": {
+          "Get a workspace with ": {
+            "$ref": "./examples/WorkspaceVirtualNetPeeringGet.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/PeeringNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/VirtualNetworkPeering"
+            }
+          },
+          "204": {
+            "description": "OK - NoContent"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "vNetPeering"
+        ],
+        "operationId": "vNetPeering_Delete",
+        "description": "Deletes the workspace vNetPeering.",
+        "x-ms-examples": {
+          "Delete a workspace": {
```

Comment:
Should it be delete a peering for a workspace? Could you share some context with us?